server/src/main/kotlin/com/xebia/functional/xef/server/http/routes/XefRoutes.kt

```diff
@@ -13,5 +12,4 @@ fun Routing.xefRoutes(
   userRoutes(UserRepositoryService(logger))
   organizationRoutes(OrganizationRepositoryService(logger))
   projectsRoutes(ProjectRepositoryService(logger))
-  tokensRoutes(TokenRepositoryService(logger))
```

Comment:
Why are we removing these routes?

Reply:
I had some struggle splitting the PR in two. I was in merge hell rebasing and cherrypicking. I tried to mitigate human mistakes. That one's on me.